tests/files/.gitignore

```diff
@@ -1,2 +1,3 @@
 *
-!.gitignore
\ No newline at end of file
+!.gitignore
+!database.sqlite
```

Comment:
I don't mind the test that creates an actual sqlite db, but I'm thinking it isn't necessary to store the generated db in the repo.